### DISCUSSION TITLE
Nuketown

### DISCUSSION THREAD

**Discussion by dmanmom** [nzp-team/nzportable#920] [General]
Can you guys make Nuketown pls

**Comment by printhelloworldasdas** [comment]
if i know how make maps i can try in my house

**Reply by printhelloworldasdas** [reply]
but first i need to know how make maps and what apps

**Reply by Peter0x44** [reply]
https://docs.nzp.gay covers most things you need to know.

**Reply by printhelloworldasdas** [reply]
thx man i will read it when im home im at the institute right now  xD

**Reply by printhelloworldasdas** [reply]
i speak spanish but i think i will understand

**Reply by dmanmom** [reply]
i'll send a pic of the map for you

**Reply by dmanmom** [reply]
here you go
![image](https://github.com/user-attachments/assets/4e63e828-b34c-40c2-b6f3-1b7304f69de1)

**Reply by printhelloworldasdas** [reply]
ok thx i will start when i knoe how download im having troubles

**Comment by ElDani123** [comment]
yeh
